examples/pytorch/rgcn/model.py

```diff
@@ -118,13 +118,14 @@ def forward(self, node_ids, node_tids, type_ids, features):
             embeddings as the input of the next layer
         """
         tsd_ids = node_ids.to(self.node_embeds.weight.device)
-        embeds = th.empty(node_ids.shape[0], self.embed_size, device=self.dev_id)
+        device_id = self.dev_id if self.dev_id >= 0 else 'cpu'
```

Comment:
We can make self.dev_id = th.device(dev_id if dev_id >= 0 else 'cpu') at __init__ function.
Then there is no need to do following changes.